.github/workflows/build.yml

```diff
@@ -34,14 +34,14 @@ jobs:
           - centos9
           - fedora39
           - fedora38
-        configure_flags: ['']
+        configure_flags: ['--enable-compatibility-mode']
         include:
           - container_tag: debian12
-            configure_flags: '--enable-debug'
+            configure_flags: '--enable-compatibility-mode --enable-debug'
           - container_tag: debian12
             # By default clang emits DWARF v5, which Valgrind cannot read yet.
             # https://github.com/llvm/llvm-project/issues/56550
-            configure_flags: 'CC=clang CXX=clang++ CFLAGS=-gdwarf-4'
+            configure_flags: '--enable-compatibility-mode CC=clang CXX=clang++ CFLAGS=-gdwarf-4'
```

Comment:
Btw. Why there are 2 `same` tags, with different configure flags, does GitHub do separate build for each?

Reply:
What's happening here is that we set up a "matrix" build with two dimensions (line 23): `container_tag` and `configure_flags`. In the standard matrix, we have a bunch of values for `container_tag` but `configure_flags` has only one value (line 37).

The `include:` on line 38 tells GitHub actions that in addition to the matrix, we want two bespoke configurations: one builds with `--enable-debug` and the other with `CC=clang`. Both use the same `container_tag`, Debian 12.

HTH